Provide access to raw response object from Faraday (#304)
crmne/ruby_llm: docs/guides/chat.md

chat.ask "What is metaprogramming in Ruby?"
```

## Raw Responses

> You can access the raw response from the API provider by setting the `raw` option to `true` when calling `ask`.

Reviewer: It seem there's no `raw: true` option.

```suggestion
You can access the raw response from the API provider with `response.raw`.
```
